### PR TITLE
DISPATCH-1555: Update doc for new console ui

### PR DESCRIPTION
**Pull request by bhardesty**

### CODE REVIEW

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 1]
```diff
@@ -24,22 +24,31 @@ under the License
 [id='monitoring-router-network-web-console'-{context}']
 = Monitoring the router network using {ConsoleName}
 
-In the web console, you use the tabs to monitor the router network.
+The web console provides several sections you can use to monitor the router network.
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: ...several sections that you can use...

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 2]
```diff
@@ -24,22 +24,31 @@ under the License
...
-[cols="30,70"]
+[cols="25,75"]
 |===
-| This tab... | Provides...
+| This section... | Provides...
 
-| `Overview` | Aggregated information about routers, addresses, links, connections, and logs.
+| Overview
+a|
+Aggregated information about the router network. This includes the following:
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: This information includes...

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 3]
```diff
@@ -24,22 +24,31 @@ under the License
...
+* Connections
+* Logs
 
-| `Topology` | A graphical view of the router network, including routers, clients, and brokers. The topology shows how the routers are connected, and how messages are flowing through the network.
+| Visualizations
+a|
+A graphical view of the router network. You can view the following types of visualizations:
 
-| `Charts` | Graphs of the information selected on the `Entities` tab.
+`Topology`:: Shows the topology of the router network, including routers, clients, and brokers. It also shows how messages are flowing through the network.
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: Topology of the router network, including routers, clients, and brokers. This visualization also shows...

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 4]
```diff
@@ -24,22 +24,31 @@ under the License
...
+* Routers
+* Addresses
+* Links
+* Connections
+* Logs
 
-| `Topology` | A graphical view of the router network, including routers, clients, and brokers. The topology shows how the routers are connected, and how messages are flowing through the network.
+| Visualizations
+a|
+A graphical view of the router network. You can view the following types of visualizations:
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: Graphical views of the router network. You can see the following types of visualizations:

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 5]
```diff
@@ -24,22 +24,31 @@ under the License
...
 
-| `Charts` | Graphs of the information selected on the `Entities` tab.
+`Topology`:: Shows the topology of the router network, including routers, clients, and brokers. It also shows how messages are flowing through the network.
+`Message flow`:: A chord diagram showing the real-time message flow by address.
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: Chord diagram showing...

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 6]
```diff
@@ -24,22 +24,31 @@ under the License
...
 
-| `Message Flow` | A chord diagram showing the real-time message flow by address.
-
-| `Schema` | The management schema that controls each of the routers in the router network.
+| Details
+| Detailed configuration information about each AMQP management entity for each router in the router network. You can view and change the configuration of any of the routers in the network.
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: Insert a pause:

Detailed configuration information about each AMQP management entity, for each router in the router network.

File: docs/books/modules/user-guide/monitoring-router-network-web-console.adoc [hunk 7]
```diff
@@ -24,22 +24,31 @@ under the License
...
+* Dashboard (shows router network statistics)
+* Routers
+* Addresses
+* Links
+* Connections
+* Logs
 
-| `Topology` | A graphical view of the router network, including routers, clients, and brokers. The topology shows how the routers are connected, and how messages are flowing through the network.
+| Visualizations
+a|
```
Comment (jbyrne-redhat) [inline docs/books/modules/user-guide/monitoring-router-network-web-console.adoc]: The three minor suggestions that follow are aimed at making all of the descriptions in this table parallel, that is, eliminate the verbs and articles that begin some descriptions.

**Comment by fgiorgetti** [comment]
@bhardesty changes look good to me. Thanks!

**Pull request closed without merging**